internal/cli/k3s/k3s.go

```diff
@@ -0,0 +1,73 @@
+package cli
+
+import (
+	"os"
+	"os/signal"
+	"path"
+	"syscall"
+
+	"github.com/spf13/cobra"
+	"github.com/weka/gohomecli/internal/cli/app"
+	"github.com/weka/gohomecli/internal/k3s"
+)
+
+func init() {
+	// by default homecli is located in /opt/wekahome/{release}/bin
+	// and bundle in /opt/wekahome/{release}/
+	bundlePath, _ := os.Executable()
+	bundlePath = path.Join(path.Dir(bundlePath), "..")
+
+	k3sInstallCmd.Flags().StringVarP(&k3sInstallConfig.Iface, "iface", "i", "", "iterface for k3s network")
+	k3sInstallCmd.Flags().StringVarP(&k3sInstallConfig.Hostname, "hostname", "n", k3s.Hostname(), "hostname for cluster")
+	k3sInstallCmd.Flags().StringSliceVar(&k3sInstallConfig.NodeIP, "ip", nil, "IP addresses for wekahome API")
+	k3sInstallCmd.Flags().StringSliceVar(&k3sInstallConfig.ExternalIP, "ip_ext", nil, "external IP addresses")
+	k3sInstallCmd.Flags().StringVar(&k3sInstallConfig.BundlePath, "bundle", bundlePath, "bundle directory where placed k3s to install")
```

Comment:
why do we need this override?

Reply:
It's used for rare scenarios when you need to pass manually (like testing). 
I'd leave it as is, but we can hide flag from users I guess if you don't like it's visibility

Reply:
yeah, hiding it is a good idea